### DISCUSSION TITLE
Log retention

### DISCUSSION THREAD

**Discussion by Hammer-Bear** [Security-Onion-Solutions/securityonion#3081] [Unsupported Versions] [answered]
Hi

So I've been asked by a client that they want 60-90 days worth of logs retention for reporting etc, currently, it does about 30 and I know it's based on the disk space/usage, is there any way to set it based on X days? or do I have to set a % of the disk and just make it a big disk to store that much?

*edit* forgot to mention this is in SO 16.04.6.4

**Comment by weslambert** [comment, marked as answer]
This forum is for Security Onion 2 only. Please use the mailing list here for SO 16.04-related questions: http://groups.google.com/group/security-onion
